plugins/jobs/web_client/views/JobDetailsWidget.js

```diff
@@ -112,6 +120,19 @@ var JobDetailsWidget = View.extend({
             startLabel: '0 s',
             endLabel: elapsed + ' s'
         }).render();
+    },
+
+    _cancelJob: function () {
+        const jobId = this.job.id;
+        restRequest({
+            path: `job/${jobId}/cancel`,
+            type: 'PUT',
+            error: null
+        }).done(_.bind(function () {
+            console.log('success');
+        }, this)).error(_.bind(function (err) {
```

Comment:
Per the changes in https://github.com/girder/girder/pull/2032 , this should be changed to `.fail`.